pkg/api/v1/atlascustomresource.go

```diff
@@ -29,6 +29,7 @@ var _ AtlasCustomResource = &AtlasStreamInstance{}
 var _ AtlasCustomResource = &AtlasStreamConnection{}
 var _ AtlasCustomResource = &AtlasSearchIndexConfig{}
 var _ AtlasCustomResource = &AtlasBackupCompliancePolicy{}
+var _ AtlasCustomResource = &AtlasPrivateEndpoint{}
 
```

Comment:
Is this here only to force an error if any of those types is not an AtlasCustomResource?

Reply:
Exactly